attributes/default.rb

```diff
@@ -85,9 +85,13 @@
 default['hops']['yarn']['vpmem_ratio']         = "50.1"
 default['hops']['yarn']['vmem_check']          = false
 end
+default['hops']['yarn']['pmem_check']                  = true
```

Comment:
this should be a string

Reply:
I also added it to metadata as it was missing